src/vue/modules/create-asset-sale/index.vue

```diff
@@ -0,0 +1,764 @@
+          hardCap: '',
+          annualReturn: '',
+          quoteAssets: [],
+        },
+        shortBlurb: {
+          saleLogo: null,
+          shortDescription: '',
+          youtubeId: '',
+          description: '',
+        },
+      },
+      isSubmitting: false,
+      currentStep: 1,
+      STEPS,
+      kvAssetTypeKycRequired: '',
+      ASSET_SUBTYPE,
+      DOCUMENT_TYPES,
+      ASSET_POLICIES,
+      CODE_MAX_LENGTH,
+      NAME_MAX_LENGTH,
+      DESCRIPTION_MAX_LENGTH,
+    }
+  },
+  validations () {
+    return {
+      form: {
+        information: {
+          name: {
+            required,
+            maxLength: maxLength(NAME_MAX_LENGTH),
+          },
+          code: {
+            required,
+            maxLength: maxLength(CODE_MAX_LENGTH),
+          },
+          assetType: {
+            required,
+          },
+          formType: {
+            required,
+          },
+        },
+        saleInformation: {
+          startTime: {
+            required,
+          },
+          endTime: {
+            required,
+            minDate: minDate(this.form.saleInformation.startTime ||
+                moment().toString()),
+          },
+          maturityDate: {
+            required: requiredUnless(function () {
+              return this.form.information.formType.value ===
+                  ASSET_SUBTYPE.bondCreation
+            }),
+            minDate: minDate(
+              this.form.information.formType.value === ASSET_SUBTYPE.bond
+                ? this.form.saleInformation.endTime : 0),
+          },
+          softCap: {
+            required,
+            amountRange: amountRange(this.minAmount, this.maxAmount),
+          },
+          hardCap: {
+            required,
+            amountRange: amountRange(this.form.saleInformation.softCap,
+              this.maxAmount),
+          },
+          annualReturn: {
+            required,
+            amountRange: amountRange('0',
+              this.maxAmount),
+          },
+          quoteAssets: {
+            requiredAtLeastOne,
+          },
+        },
+        shortBlurb: {
+          saleLogo: {
+            required,
+          },
+          shortDescription: {
+            required,
+            maxLength: maxLength(DESCRIPTION_MAX_LENGTH),
+          },
+        },
+      },
+    }
+  },
+  computed: {
+    formTypes () {
+      return [
+        {
+          label: 'create-asset-sale-form.bond-creation',
+          value: ASSET_SUBTYPE.bond,
+        },
+        {
+          label: 'create-asset-sale-form.property-purchase',
+          value: ASSET_SUBTYPE.share,
+        },
+      ]
+    },
+    assetTypes () {
+      return [
+        {
+          label: 'create-asset-sale-form.asset-type-not-required-kyc',
+          value: '0',
+        },
+        {
+          label: 'create-asset-sale-form.asset-type-required-kyc',
+          value: String(this.kvAssetTypeKycRequired),
+        },
+      ]
+    },
+    baseAssets () {
```

Comment:
When you will move the ‘assets’ to the getters, you can define baseAssets and statsQuoteAsset also as getters